src/ProtobufServer/ecdar_requests/send_query.rs

```diff
@@ -159,8 +161,36 @@ fn convert_ecdar_result(query_result: &QueryResult) -> Option<ProtobufResult> {
             refine::RefinementResult::Failure(failure) => convert_refinement_failure(failure),
         },
 
-        QueryResult::Reachability(_) => {
-            unimplemented!("Not implemented, but should be implemented");
+        QueryResult::Reachability(path) => {
+            let protoPath = TransitionID::split_into_component_lists(
+                &path
+                    .path
+                    .as_ref()
+                    .unwrap()
```

Comment:
Just to be sure, there is no way this can cause a panic?

Reply:
Internal branch XD but thx well look at it :)